addons/website/views/snippets/s_image_comparison.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="s_image_comparison" name="Image Comparison">
+        <section class="s_image_comparison pt24 pb24 o_not_editable" contenteditable="false">
+            <div class="o_container_small">
+                <div class="col-12">
+                    <figure class="figure">
+                        <img class="o_media_right" src="/web/image/website.s_image_comparison_1"/>
+                        <img class="o_media_left" src="/web/image/website.s_image_comparison_2"/>
+                        <input type="range" min="0" max="100" value="50"/>
+                        <div class="o_description o_right_description o_not_editable" contenteditable="false">
+                            <span contenteditable="true">AFTER</span>
```

Comment:
Same: contenteditable="false" probably not needed. I would also remove the "true" one and set it manually in JS via the `onBuilt` method.